.github/workflows/cmake.yml

```diff
@@ -134,22 +133,21 @@ jobs:
           path: ${{ github.workspace }}/Build/CLDR
           key: UnicodeLocale-${{ hashFiles('Meta/CMake/unicode_data.cmake') }}
       - name: Create build environment with extra debug options
-        working-directory: ${{ github.workspace }}/Build
         # Build the entire project with all available debug options turned on, to prevent code rot.
         # However, it is unweildy and slow to run tests with them enabled, so we will build twice.
-        run: cmake .. -GNinja -DSERENITY_ARCH=${{ matrix.arch }} -DBUILD_LAGOM=ON -DENABLE_ALL_DEBUG_FACILITIES=ON -DENABLE_PCI_IDS_DOWNLOAD=OFF -DENABLE_USB_IDS_DOWNLOAD=OFF -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10
+        run: cmake -S ./Meta/CMake/Superbuild -B ./Build/superbuild -GNinja -DSERENITY_ARCH=${{ matrix.arch }} -DBUILD_LAGOM=ON -DENABLE_ALL_DEBUG_FACILITIES=ON -DENABLE_PCI_IDS_DOWNLOAD=OFF -DENABLE_USB_IDS_DOWNLOAD=OFF -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10
```

Comment:
I think I'm missing a step here, because right after creating the superbuild directory, there isn't a CMakeCache.txt file in the child build binary directories.

Reply:
Right, you caught me with my pants down. A build must happen first before those directories are populated. Now my initial suggestion just makes me look silly.

I guess this can be resolved and continued in the [discussion for Superbuild/CMakeLists.txt](https://github.com/SerenityOS/serenity/pull/9297#discussion_r697862017).